include/deal.II/numerics/data_out.h

```diff
@@ -274,14 +274,14 @@ class DataOut : public DataOut_DoFData<DH, DH::dimension, DH::space_dimension>
    * pair that is locally owned. If this object operates on a non-distributed
    * triangulation, the result equals what first_cell() returns.
    */
-  cell_iterator first_locally_owned_cell ();
+  virtual cell_iterator first_locally_owned_cell ();
```

Comment:
Good catch, I will remove this for now because it is not needed. It was actually quite useful for a test (derive from DataOut to create a LevelDataOut class).